semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/nodemethods/NodeMethods.scala

```diff
@@ -33,4 +34,7 @@ class NodeMethods(node: Node) {
         Nil
     }
 
+  def reference: Option[nodes.StoredNode] =
```

Comment:
This is only used in ReachingDefPass.scala. Instead of putting it here in this super generic place please put it into ReachingDefPass.scala to avoid cluttering the method name space on Node.